src/main/java/com/cburch/logisim/circuit/appear/DefaultAppearance.java

```diff
@@ -33,26 +31,7 @@ public static List<CanvasObject> build(
   }
 
   static void sortPinList(List<Instance> pins, Direction facing) {
-    // Remove any clock pins and then add them to the end after sorting
-    ArrayList<Instance> clockPins = new ArrayList<>();
-    for (var it = pins.iterator(); it.hasNext(); ) {
-      var pin = it.next();
-      if (Pin.FACTORY.isClockPin(pin)) {
-        clockPins.add(pin);
-        it.remove();
-      }
-    }
-
-    if (facing == Direction.NORTH || facing == Direction.SOUTH) {
-      Location.sortHorizontal(pins);
-      Location.sortHorizontal(clockPins);
-    }
-    else {
-      Location.sortVertical(pins);
-      Location.sortVertical(clockPins);
-    }
-
-    // Clock pins always come last
-    pins.addAll(clockPins);
+    if (facing == Direction.NORTH || facing == Direction.SOUTH) Location.sortHorizontal(pins);
+    else Location.sortVertical(pins);
```

Comment:
A multi-line if should have {}. An else should always appear as "} else {".

Reply:
Sorry my bet, I just copied the old code; fixed